<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="4" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-07T13:34:01Z" name="Suite" finished-at="2022-08-07T13:34:29Z" duration-ms="28320">
    <groups>
    </groups>
    <test started-at="2022-08-07T13:34:01Z" name="Test1" finished-at="2022-08-07T13:34:28Z" duration-ms="27350">
      <class name="testng_packages.Testcase_Datadriven">
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:testng_packages.Testcase_Datadriven@3148f668]" started-at="2022-08-07T19:04:02Z" name="login" data-provider="dp" finished-at="2022-08-07T19:04:12Z" duration-ms="10169" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[add]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122342.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:testng_packages.Testcase_Datadriven@3148f668]" started-at="2022-08-07T19:04:13Z" name="login" data-provider="dp" finished-at="2022-08-07T19:04:15Z" duration-ms="2608" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[adad]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[asddas]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:testng_packages.Testcase_Datadriven@3148f668]" started-at="2022-08-07T19:04:15Z" name="login" data-provider="dp" finished-at="2022-08-07T19:04:18Z" duration-ms="2915" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[aasa]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1232.0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:testng_packages.Testcase_Datadriven@3148f668]" started-at="2022-08-07T19:04:18Z" name="login" data-provider="dp" finished-at="2022-08-07T19:04:28Z" duration-ms="9816" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- testng_packages.Testcase_Datadriven -->
    </test> <!-- Test1 -->
    <test started-at="2022-08-07T13:34:28Z" name="test2" finished-at="2022-08-07T13:34:28Z" duration-ms="309">
      <class name="testng_packages.Testcases_login_dashboard">
        <test-method signature="pendingleave()[pri:0, instance:testng_packages.Testcases_login_dashboard@20ca951f]" started-at="2022-08-07T19:04:28Z" name="pendingleave" finished-at="2022-08-07T19:04:28Z" duration-ms="4" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testng_packages.Testcases_login_dashboard.pendingleave(Testcases_login_dashboard.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pendingleave -->
      </class> <!-- testng_packages.Testcases_login_dashboard -->
    </test> <!-- test2 -->
    <test started-at="2022-08-07T13:34:28Z" name="test3" finished-at="2022-08-07T13:34:29Z" duration-ms="343">
      <class name="testng_packages.Testcases_AdminUsers">
        <test-method signature="userdetails()[pri:0, instance:testng_packages.Testcases_AdminUsers@44d52de2]" started-at="2022-08-07T19:04:28Z" name="userdetails" finished-at="2022-08-07T19:04:28Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testng_packages.Testcases_AdminUsers.userdetails(Testcases_AdminUsers.java:71)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userdetails -->
      </class> <!-- testng_packages.Testcases_AdminUsers -->
    </test> <!-- test3 -->
    <test started-at="2022-08-07T13:34:29Z" name="test4" finished-at="2022-08-07T13:34:29Z" duration-ms="246">
      <class name="testng_packages.Testcase_Datadriven">
        <test-method is-config="true" signature="launchbrowser()[pri:0, instance:testng_packages.Testcase_Datadriven@3148f668]" started-at="2022-08-07T19:03:57Z" name="launchbrowser" finished-at="2022-08-07T19:04:01Z" duration-ms="3705" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchbrowser -->
        <test-method is-config="true" signature="driverclose()[pri:0, instance:testng_packages.Testcase_Datadriven@3148f668]" started-at="2022-08-07T19:04:29Z" name="driverclose" finished-at="2022-08-07T19:04:30Z" duration-ms="808" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverclose -->
      </class> <!-- testng_packages.Testcase_Datadriven -->
      <class name="testng_packages.Testcase_myinfo">
        <test-method signature="imageupload()[pri:0, instance:testng_packages.Testcase_myinfo@3688eb5b]" started-at="2022-08-07T19:04:29Z" name="imageupload" finished-at="2022-08-07T19:04:29Z" duration-ms="1" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testng_packages.Testcase_myinfo.imageupload(Testcase_myinfo.java:19)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- imageupload -->
      </class> <!-- testng_packages.Testcase_myinfo -->
    </test> <!-- test4 -->
  </suite> <!-- Suite -->
</testng-results>
